miden/src/cli/debug/executor.rs

```diff
@@ -180,23 +180,36 @@ impl DebugExecutor {
 
     /// print help message
     fn print_help() {
-        let message = "---------------------------------------------------------\n\
+        let message = "---------------------------------------------------------------------\n\
             Miden Assembly Debug CLI\n\
-            ---------------------------------------------------------\n\
-            !next        steps to the next clock cycle\n\
-            !play        executes program until completion or failure\n\
-            !play.n      executes n clock cycles\n\
-            !prev        steps to the previous clock cycle\n\
-            !rewind      rewinds program until beginning\n\
-            !rewind.n    rewinds n clock cycles\n\
-            !print       displays the complete state of the virtual machine\n\
-            !stack       displays the complete state of the stack\n\
-            !stack[i]    displays the stack element at index i\n\
-            !mem         displays the complete state of memory\n\
-            !mem[i]      displays memory at address i\n\
-            !clock       displays the current clock cycle\n\
-            !quit        quits the debugger\n\
-            !help        displays this message";
+            ---------------------------------------------------------------------\n\
+            next               steps to the next clock cycle\n\
+            next <c>           steps to the next `c` clock cycles\n\
+            continue           executes program until completion or failure\n\
+            back               rewinds `1` clock cycles\n\
+            back <c>           rewinds `c` clock cycles\n\
+            rewind             rewinds program until beginning\n\
+            print              displays the complete state of the virtual machine\n\
+            print mem          displays the complete state of memory\n\
+            print mem <i>      displays memory at address `i`\n\
+            print stack        displays the complete state of the stack\n\
+            print stack <i>    displays the stack element at index `i`\n\
+            clock              displays the current clock cycle\n\
+            quit               quits the debugger\n\
+            help               displays this message\n\
+            \n\
+            The following mappings are also available:\n\
```

Comment:
I wonder if we could format the short form commands in line.  Maybe we create an extra column for the short form and add titles to the columns something like:
```
Long                      |      Short      |                  Description
----------------------------------------------------------------------------------------------
print mem <i>                     p m <i>             displays memory at address `i`\n\
```

Reply:
Printing tables on CLI is not that trivial as it depends on the font of choice, the characteristics of the terminal, etc.

We do have some crates out there that will take care of all of that for us, but I wonder if it's worthy to add another dependency. Wdyt if we leave this as open issue for future improvement?

If we decide to include in this PR, I think the best call is https://crates.io/crates/prettytable-rs

Reply:
yeah I think this is more of a nice to have then a hard requirement.  It doesn't really affect UX, it just makes it a little more pretty but that not hugely important.  I don't think it's worth spending time on now if it's not simple.